pkg/connector/client.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/gotd/td/telegram/updates"
 	"github.com/gotd/td/tg"
 	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
```

Comment:
There's a global log somewhere 🙀

Reply:
https://github.com/beeper/pre-commit-go/commit/f7add0762927d04f68ca00641464c04f03fa5d8d

and

https://github.com/mautrix/telegramgo/commit/0046975aa5508c81c778c421154b3137f3ba8534